pkg/client/status_reporter.go

```diff
@@ -136,7 +137,7 @@ func (r *StatusReporter) Get() (*v1.ClusterOperator, error) {
 	return r.client.Get(context.TODO(), r.clusterOperatorName, metav1.GetOptions{})
 }
 
-func (r *StatusReporter) SetFailed(statusErr error, reason string) error {
+func (r *StatusReporter) SetFailed(statusErr []error, reason string) error {
```

Comment:
Instead of changing `SetFailed()` signature and passing it a slice of errors, it would feel more natural to me to build the final `error` in `Operator.sync()`.

Reply:
Done